### ISSUE TITLE
WCAG20-TECHS ARIAパート コメント(8) - ARIA6 例1 HTML4とXHTML1.0文書の限定の必要性

### ISSUE THREAD

**Issue by momdo** [momdo/momdo.github.io#20]
HTML5だとこうは記述しないだろう。ARIA11 例1-3にも同様な記述あり。

**Comment by momdo** [comment]
ARIA11 例1-3にも同様な記述あり。

**Comment by momdo** [comment]
Why not HTML5?
Example1 of ARIA6 and example 1-3 of ARIA11 say "HTML4 or XHTML 1.0 document", why?

**Comment by makotom** [comment]
こんなんでどうでしょう:

ARIA6 and ARIA11: Content updates for HTML5 required
Example1 of ARIA6 and example 1-3 of ARIA11 say "HTML4 or XHTML 1.0 document", where we have HTML5 already. They should be updated.
Using HTML5, we can write these examples more easily and clearly, along with Implicit ARIA semantics. Especially, we should review description section of ARIA11 from the viewpoint of ARIA in HTML spec: we have nav element for the purpose, which should replace the navigation role unless otherwise there is a specific reason to rely on WAI-ARIA.

**Comment by momdo** [comment]
@makotom
ありがとうございます。
issue 155として報告しました。

**Comment by momdo** [comment]
2月末にwon't fix扱い（でも閉じてない）が反論する気力も無いので閉じておく。
